docs/Advanced/SQL Queries.md

```diff
@@ -0,0 +1,72 @@
+---
+publish: true
+---
+
+# SQL Queries
+
+<span class="related-pages">#css</span>
```

Comment:
Could you please add a note somewhere in this doc about how to add comments in your queries?